### PR TITLE
Add HttpClient functions

### PR DESCRIPTION
**Pull request by mariajdab**
## What is the problem/goal being addressed?
Add functions to `LibHttpClient.fs` and the test cases
## What is the solution to this problem?
Add functions to `LibHttpClient.fs` and the test cases
## How are you sure this works/how was this tested?
Test cases passed
## Has this information been included in the comments?
Yes

### CODE REVIEW

File: fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs [hunk 1]
```diff
@@ -19,109 +19,98 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
...
+      returnType = TDict varA
+      description = "Returns an object with 'Authorization' set to the passed token"
+      fn =
+        (function
+        | _, [ DStr token ] ->
+            let authString =
+              System.Text.Encoding.UTF8.GetString(
+                Array.append
+                  (System.Text.Encoding.UTF8.GetBytes "Bearer ")
+                  (System.Text.Encoding.UTF8.GetBytes token)
```
Comment (pbiggar) [inline fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs:95]: I think you should use string append here instead of converting to bytes.

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks great. I change noted in text. Can you also change all the signatures to `TDict TStr` as they all return dicts of strings.

File: fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs (hunk 1 above)
Comment (mariajdab) [reply fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs:95]: Thank you! I will add the changes

File: fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs [hunk 2]
```diff
@@ -83,17 +83,12 @@ let fns : List<BuiltInFn> =
...
       fn =
         (function
         | _, [ DStr token ] ->
-            let authString =
-              System.Text.Encoding.UTF8.GetString(
-                Array.append
-                  (System.Text.Encoding.UTF8.GetBytes "Bearer ")
-                  (System.Text.Encoding.UTF8.GetBytes token)
-              )
+            let authString = [ "Bearer "; token ] |> String.concat ""
```
Comment (pbiggar) [inline fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs:91]: You can use `"Bearer " + token`

**Pull request merged**